vendor/github.com/stripe/goproxy/https.go

```diff
@@ -543,11 +544,19 @@ func (proxy *ProxyHttpServer) connectDialProxyWithContext(ctx *ProxyCtx, proxyHo
 		c = tls.Client(c, proxy.Tr.TLSClientConfig)
 	}
 
+	hdr := make(http.Header)
+
+	// Add proxy authentication header if needed
+	auth := proxyURL.User.String()
```

Comment:
I think we need a nil check here before calling `String()`.

Reply:
Will fix in `goproxy`!